hack/images

```diff
@@ -7,7 +7,7 @@ PUSH=$3
 . $(dirname $0)/util
 set -eu -o pipefail
 
-: ${PLATFORMS=linux/amd64}
+: ${PLATFORMS=local}
```

Comment:
That's not correct as there are no darwin images etc. I think the previous iteration of this PR was ok, just with the `=` flag.

Reply:
Oh ok get it was confused with the `local` behavior there, that makes sense now.